client/src/components/PublicPages/AuditPlanner/sampleSizes.ts

```diff
@@ -40,6 +41,5 @@ export const useSampleSizes = (
       onError: showToastOnError
         ? error => toast.error(error.message)
         : () => {}, // eslint-disable-line @typescript-eslint/no-empty-function
-      staleTime: Infinity, // Allow use of cached query results
```

Comment:
Now that we're precomputing for all risk limit percentages, caching is no longer as important / beneficial (we'd just be caching sample sizes across different edits of election results)

Reply:
Makes sense to me!